apps/server/src/modules/tool/context-external-tool/service/tool-version-service.ts

```diff
@@ -1,74 +1,57 @@
-import { Inject } from '@nestjs/common';
 import { Injectable } from '@nestjs/common/decorators/core/injectable.decorator';
 import { ValidationError } from '@shared/common';
 import {
 	ContextExternalToolConfigurationStatus,
 	ToolParameterValueMissingLoggableException,
 } from '../../common/domain';
-import { CommonToolService, CommonToolValidationService } from '../../common/service';
+import { CommonToolValidationService } from '../../common/service';
 import { ExternalTool } from '../../external-tool/domain';
 import { SchoolExternalTool } from '../../school-external-tool/domain';
-import { IToolFeatures, ToolFeatures } from '../../tool-config';
 import { ContextExternalTool } from '../domain';
 
 @Injectable()
 export class ToolVersionService {
```

Comment:
We seem to still have some validation in this file, but there is no version in tools anymore, right? Why do we still have a tool version service? Maybe we could rename the file or move the validation logic somewhere else... 🤷‍♂️

Reply:
Something without the name "version" in it 😅 ToolConfigurationStatusService? 🤷‍♂️